packages/clay-core/src/language-picker/LanguagePicker.tsx

```diff
@@ -133,18 +133,20 @@ const getTranslationLabel = ({
 	let displayType: DisplayType = 'warning';
 	let label: string = messages.untranslated;
 
-	if (localeId === defaultLocaleId) {
-		displayType = 'info';
-		label = messages.default;
-	} else if (translation) {
+	if (translation) {
 		const {total, translated} = translation;
 
-		if (total && total === translated) {
-			displayType = 'success';
-			label = messages.translated;
-		} else {
-			displayType = 'secondary';
-			label = sub(messages.translating, [translated, total]);
+		if (total !== 0) {
```

Comment:
This logic doesn't seem right. This means the only way to display the untranslated message would be if total is 0. But I think total refers to "total available translations", so this isn't correct. It should probably be `if (translated !== 0)` since translated refers to which of the available translations have been translated.